contributors/devel/automation.md

```diff
@@ -38,37 +38,33 @@ The status of the submit-queue is [online.](http://submit-queue.k8s.io/)
 The submit-queue lists what it believes are required on the [merge requirements tab](http://submit-queue.k8s.io/#/info) of the info page. That may be more up to date.
 
 A PR is considered "ready for merging" if it matches the following:
-  * The PR must have the label "cla: yes" or "cla: human-approved"
-  * The PR must be mergeable. aka cannot need a rebase
-  * All of the following github statuses must be green
-     * Jenkins GCE Node e2e
-     * Jenkins GCE e2e
-     * Jenkins unit/integration
+  * The PR must have the label `cncf-cla: yes` or `cla: human-approved`
+  * The PR must be mergeable, aka cannot need a rebase
+  * All of the [Required Github CI Tests](https://github.com/kubernetes/test-infra/blob/master/mungegithub/submit-queue/deployment/kubernetes/configmap.yaml#L15) must be green
   * The PR cannot have any prohibited future milestones (such as a v1.5 milestone during v1.4 code freeze)
-  * The PR must have the "lgtm" label. The "lgtm" label is automatically applied
-    following a review comment consisting of only "LGTM" (case-insensitive)
-  * The PR must not have been updated since the "lgtm" label was applied
-  * The PR must not have the "do-not-merge" label
+  * The PR must have the `lgtm` label. The `lgtm` label is automatically applied
+    following a review comment consisting of `/lgtm` (case-insensitive) in a single line
+  * The PR must not have been updated since the `lgtm` label was applied
+  * The PR must have the `approved` label. The `approved` label is automatically applied after all approvers have approved the PR 
+    by a comment consisting of `/approve` (case-insensitive) in a single line
+  * The PR must not have the `do-not-merge` label or any of other `do-not-merge/*` labels, e.g., `do-not-merge/release-note-label-needed`
 
 ### Merge process
 
 Merges _only_ occur when the [critical builds](http://submit-queue.k8s.io/#/e2e)
```

Comment:
There is no e2e tab anymore and no such thing as critical builds. All the ci builds are non-blocking.